### ISSUE TITLE
Skype can't connect via corporate MitM proxy

### ISSUE THREAD

**Issue by yrro** [flatpak/flatpak#2297]
## Linux distribution and version
Debian unstable
## Flatpak version
1.0.4
## Description of the problem
Skype says "Unable to sign in. Please check your internet connection and try again".
## Steps to reproduce
Run Skype behind a TLS-interception middlebox

Naturally Skype itself logs nothing of any use, but using Wireshark I can see that Skype is successfully connecting out to 157.55.134.136 and performing a TLS handshake, then closing the connection.

The certificate it receives is signed by our TLS-interception middlebox. I presume Skype is detecting the man-in-the-middle attack and refusing to continue.

I've added our corporate CA certificate to my machine's trust store (on Debian, this shows up in /etc/ssl/certs) but I presume that inside a flatpak application's namespace I'm getting the runtime's certificate list...

**Comment by TingPing** [comment]
I don't believe OpenSSL in Flatpak can use host certs, only libraries that support pkcs11: https://github.com/flatpak/flatpak/pull/1757

**Comment by baptx** [comment]
I have the same problem with Telegram desktop, I cannot use the Flatpak version with mitmproxy. So the only solution to intercept HTTP requests is to not use Flatpak?

**Comment by Flashwalker** [comment]
> So the only solution to intercept traffic or use a SOCKS proxy is to not use Flatpak?

Yeah. Flatpak sucks with proxy and vpn.

**Comment by yrro** [comment]
Skype is long dead. #2721 covers the need for apps to trust CA certificates installed on the host. Closing.
